### PR TITLE
Optimize asset load

### PR DESCRIPTION
**Pull request by uniboi**
some quick changes I made to not watch the northstar 1k resolution logo download just to get downscaled to 192x192

* loads different resolutions on mobile / desktop
* mobile images have more detail to look sharp on smaller viewports
* all images downscaled to their actual used size and compressed
* only load the background video on desktop. There's no point to download an additional 8mb on mobile devices for something as trivial as this. Also the video lagging my phone out.

### CODE REVIEW

**Comment by x3Karma** [comment]
you need to have this playtested first

**Comment by ASpoonPlaysGames** [comment]
"playtested"

**Review by pg9182 — CHANGES_REQUESTED** [review]
Do not use JavaScript for setting images; use the [`<picture>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) element.

File: script/index.js [hunk 1]
```diff
@@ -1,21 +1,48 @@
...
...dding\x20docs\x20(', 'log', '7223992BtxUJz', '2064209JWVbyU', '286914NYeCpq', '8653536OttIGJ']; _0xe6ea = function () { return _0x640589; }; return _0xe6ea(); } var _0x3f0821 = _0x56b1; function _0x56b1(_0x2b5818, _0x5491bc) { var _0xe6ea0d = _0xe6ea(); return _0x56b1 = function (_0x56b1ff, _0x153296) { _0x56b1ff = _0x56b1ff - 0x14f; var _0xa5bd01 = _0xe6ea0d[_0x56b1ff]; return _0xa5bd01; }, _0x56b1(_0x2b5818, _0x5491bc); } (function (_0x195461, _0x4a80ec) { var _0x3ee3f4 = _0x56b1, _0x5badab = _0x195461(); while (!![]) { try { var _0xfe9d3f = parseInt(_0x3ee3f4(0x158)) / 0x1 * (parseInt(_0x3ee3f4(0x155)) / 0x2) + -parseInt(_0x3ee3f4(0x150)) / 0x3 + -parseInt(_0x3ee3f4(0x15a)) / 0x4 * (-parseInt(_0x3ee3f4(0x157)) / 0x5) + -parseInt(_0x3ee3f4(0x156)) / 0x6 + -parseInt(_0x3ee3f4(0x154)) / 0x7 + -parseInt(_0x3ee3f4(0x153)) / 0x8 * (-parseInt(_0x3ee3f4(0x15d)) / 0x9) + parseInt(_0x3ee3f4(0x14f)) / 0xa; if (_0xfe9d3f === _0x4a80ec) break; else _0x5badab['push'](_0x5badab['shift']()); } catch (_0x270bc5) { _0x5badab['push'](_0x5badab['shift']()); } } }(_0xe6ea, 0xe6076), console[_0x3f0821(0x152)](_0x3f0821(0x159) + location[_0x3f0821(0x15c)] + _0x3f0821(0x151) + _0x3f0821(0x15b) + ')'));
```
Comment (emma-miler) [inline script/index.js:15]: This can realistically just be removed.

File: script/index.js [hunk 2]
```diff
@@ -1,21 +1,48 @@
...
+    document.getElementById("browser-globe").src = `/assets/globe${desktop ? "" : "-m"}.png`
+
+    if (desktop) {
+        // video is lagging on mobile and 8mb is quite large so this is desktop only
+        document.getElementById("lazyvideo").src = "/assets/NorthstarPromoVid.mp4"
+    }
+}
+
+// images set by this won't load on browser with javascript disabled
+setPlatformSources(!navigator.userAgent.match(/iphone|android/ig))
```
Comment (emma-miler) [inline script/index.js:48]: Please dont do this. We have `<picture>` for a reason

File: index.html [hunk 3]
```diff
@@ -77,51 +77,50 @@
...
                     <a ondragstart="return false;" href="https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/manual-installation" target="_blank" class="button"><img src="/assets/icon_wiki.svg"/><span>Guide</span></a>
                 </div>
             </div>
         </div>
     </div>
     <div class="pane">
         <div class="section">Support</div>
-        <p style="text-align: center" class="supporttext">If you're having issues with running or setting up the Northstar client or are thinking of running a dedicated server you can find help and troubleshooting tips on the wiki and the Discord server:</p>
+        <p style="text-align: center" class="supporttext">
+            If you're having issues with running or setting up the Northstar client or are thinking of running a dedicated server you can find help and troubleshooting tips on the wiki and the Discord server:</p>
```
Comment (emma-miler) [inline index.html:123]: Whitespace changes?

File: index.html [hunk 4]
```diff
@@ -52,19 +52,19 @@
...
                 <span class="northstartext">NORTHSTAR</span>
             </div>
             <p class="northstardescription"><span class="tffont">titanfall<span class="tfdivider"></span> 2</span> modding and custom server framework</p>
             <div class="centerbuttons">
                 <a ondragstart="return false;" href="/discord" target="_blank" class="button big"><img src="/assets/icon_discord.svg" alt="Discord Server"/><span>Discord</span></a>
                 <a href="/wiki" class="button big"><img src="/assets/icon_wiki.svg" alt="Northstar Wiki"/>Wiki</a>
-                <a href="/github" class="button big"><img src="/assets/github.png" alt="Modding Documentation"/>GitHub</a>
-                <a href="/servers" class="button big"><img src="/assets/globe.png" alt="Modding Documentation"/>Server Browser</a>
+                <a href="/github" class="button big"><img src="" class="gh_img" alt="Github"/>GitHub</a>
+                <a href="/servers" class="button big"><img id="browser-globe" src="" alt="Server Browser"/>Server Browser</a>
```
Comment (emma-miler) [inline index.html:67]: This goes for all, but class names should be expanded

**Review by emma-miler — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (uniboi) [reply index.html:67]: cringe

**Comment by BigSpice** [comment]
bump

**Comment by pg9182** [comment]
I'm going to close this for now, as there are many conflicts, no updates, and it mostly needs to be rewritten to use `<picture>` anyways.

**Pull request closed without merging**